Example/Messaging/Tests/FIRMessagingTest.m

```diff
@@ -22,6 +22,7 @@
 #import <FirebaseInstanceID/FirebaseInstanceID.h>
 #import <FirebaseAnalyticsInterop/FIRAnalyticsInterop.h>
 #import <FirebaseMessaging/FIRMessaging.h>
+#import <GoogleUtilities/GULUserDefaults.h>
```

Comment:
This import doesn't seem to be needed since the class uses NSUserDefaults directly